reactjs/components/atoms/Paragraph/List/List.js

```diff
@@ -37,8 +37,8 @@ class List extends React.Component {
             <Wrapper>
               {list.map((item, index) => (
                 type === 'list_checkbox' ?
-                  <Checkbox key={index} label={item}/> :
-                  <li key={index}><span>{item}</span></li>
+                  <Checkbox key={this.props.id + '_' + index} label={item}/> :
+                  <li key={this.props.id + '_' + index}><span>{item}</span></li>
```

Comment:
@andreityuhai yeah! Now, to `id` prop from `<Checkbox />` component. You can pass exactly the same unique value and remove md5 from `CheckBox/index.js` as well. 

P.S. Small hint: you can extract props into variables instead of `this.props.id`. See how `list` and  `type` are extracted: https://github.com/anu-lms/Anu-LMS/blob/master/reactjs/components/atoms/Paragraph/List/List.js#L22